README.markdown

```diff
@@ -15,38 +7,39 @@ Authority will work fine with a standalone app or a single sign-on system. You c
 If you're using it with Rails controllers, it requires that you already have some kind of user object in your application, accessible via a method like `current_user` (configurable).
 
 [![Gem Version](https://badge.fury.io/rb/authority.png)](https://rubygems.org/gems/authority)
```

Comment:
Would be good to update this when we know where we'll be hosting our version of the gem.